### PR TITLE
Updating the Screensharing Mode policy documentation

### PR DESCRIPTION
**Pull request by nej-nej**
Updating the Screensharing Mode policy documentation to improve clarity

### CODE REVIEW

File: Teams/settings-policies-reference.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ Meeting policies are used to control what features are available in meetings org
 | Setting | Default | Description |
 |:-----|:-----|:-----|
 |Who can present|Everyone|Controls who can be a presenter in Teams meetings. Organizers and co-organizers can change this when they set up Teams meetings.|
-|Screen sharing mode|Entire screen|Controls whether desktop and window sharing is allowed in the user's meeting. Read more on how to [Configure desktop sharing in Microsoft Teams](configure-desktop-sharing.md).|
+|Screen sharing mode|Entire screen|Controls whether a user is allowed to share desktop and window in a Teams meeting. Read more on how to [Configure desktop sharing in Microsoft Teams](configure-desktop-sharing.md).|
```
Comment (JohanFreelancer9) [inline Teams/settings-policies-reference.md:237]:
```suggestion
|Screen sharing mode|Entire screen|Controls whether a user is allowed to share a desktop and window in a Teams meeting. Read more on how to [Configure desktop sharing in Microsoft Teams](configure-desktop-sharing.md).|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nej-nej, kindly commit the suggested change. Many thanks.

**Comment by nej-nej** [comment]
Committing change as per suggestions

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@mkbond007 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**